### PR TITLE
Fix get-disk documentation

### PR DESCRIPTION
**Pull request by jzabroski**
My NOTE is basically verbatim quote of the following answer by TEchNet user AnnaWY

https://social.technet.microsoft.com/Forums/windowsserver/en-US/cd7c0327-3fe9-45fc-a177-5a9927d468f3/does-the-getdisk-funtion-only-return-basic-disks?forum=winserverpowershell

See also: https://blogs.technet.microsoft.com/tip_of_the_day/2014/04/25/tip-of-the-day-dynamic-disks-and-windows-powershell/ Robert Mitchell [MSFT] saying Dynamic disks are deprecated.  ~~I'm separately curious if Microsoft is changing their stance on this considering the now-largest deployment of Microsoft OS licenses in Amazon and they use Dynamic Disks, ugh, everywhere.~~ [redacted: AWS Support explained to me that EBS uses whatever the OS is configured to use.  They do not care how you configure the disks, they just present the media to the OS and you configure it.]

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: docset/windows/storage/get-disk.md [hunk 1]
```diff
@@ -23,6 +23,8 @@ ms.assetid: 3929EB27-1365-42CA-8ACF-68B364C3599F
 ## SYNOPSIS
 Gets one or more disks visible to the operating system.
 
+NOTE: This cmdlet returns physical disk objects like basic disks and partitioned drive partitions.  Dynamic disks can span multiple pieces of physical media, and so they will not be returned by Get-Disk. For more information, please see: https://docs.microsoft.com/en-us/windows/desktop/FileIO/basic-and-dynamic-disks
```
Comment (nenonix) [inline docset/windows/storage/get-disk.md]:
```suggestion
NOTE: This cmdlet returns physical disk objects like basic disks and partitioned drive partitions.  Dynamic disks can span multiple pieces of physical media, so they will not be returned by Get-Disk. For more information, see [Basic and Dynamic Disks](https://docs.microsoft.com/windows/desktop/FileIO/basic-and-dynamic-disks).
```

**Comment by e0i** [comment]
@jzabroski Any updates with the suggested revisions? We would like to progress this PR into technical review. Thanks.

**Comment by jzabroski** [comment]
I thought the reason dynamic disks are not returned is because they're deprecated.  The current wording suggests other storage media like Windows Storage Spaces is also not supported if the Storage Space spans multiple disks.

This was explained to me in a separate conversation with AWS Support.

**Comment by jzabroski** [comment]
I accepted @nenonix changes.  I think the explanation provided on Microsoft TechNet and MSDN by Microsoft employees who wrote the filesystem logic trumps AWS Support. :)

**Comment by jzabroski** [comment]
Updated original comment to clarify I was incorrect about AWS using Dynamic Volumes as well.

This PR is good to go.

**Comment by e0i** [comment]
@dariomws306 @get-itips @tseward

Peer review needed for this PR. Thanks.

**Comment by dariomws306** [comment]
Looks ok.

But I think we should use the standard format for notes in docs. You should use:

> [!NOTE]
> This cmdlet returns physical disk objects like basic disks and partitioned drive partitions.  Dynamic disks can span multiple pieces of physical media, so they will not be returned by Get-Disk. For more information, see [Basic and Dynamic Disks](https://docs.microsoft.com/windows/desktop/FileIO/basic-and-dynamic-disks).

Thanks!

**Comment by dariomws306** [comment]
> [!NOTE]
>

**Comment by get-itips** [comment]
Agree with @dariowms306

File: docset/windows/storage/get-disk.md [hunk 2]
```diff
@@ -23,6 +23,8 @@ ms.assetid: 3929EB27-1365-42CA-8ACF-68B364C3599F
 ## SYNOPSIS
 Gets one or more disks visible to the operating system.
 
+NOTE: This cmdlet returns physical disk objects like basic disks and partitioned drive partitions.  Dynamic disks can span multiple pieces of physical media, so they will not be returned by Get-Disk. For more information, see [Basic and Dynamic Disks](https://docs.microsoft.com/windows/desktop/FileIO/basic-and-dynamic-disks).
```
Comment (nenonix) [inline docset/windows/storage/get-disk.md]:
```suggestion
> [!NOTE]
> This cmdlet returns physical disk objects like basic disks and partitioned drive partitions.  Dynamic disks can span multiple pieces of physical media, so they will not be returned by Get-Disk. For more information, see [Basic and Dynamic Disks](https://docs.microsoft.com/windows/desktop/FileIO/basic-and-dynamic-disks).
```

**Comment by e0i** [comment]
@jzabroski Please follow-up with the suggested NOTE formatting so that the PR can proceed into merge. Thanks.

**Comment by jzabroski** [comment]
@e0i  THanks for the ping - done.

**Comment by e0i** [comment]
@tseward Any concluding thoughts with the PR? Thanks.

**Comment by tseward** [comment]
Looks good!

**Comment by e0i** [comment]
@AndreaBarr The community-contributed PR that has passed both copy and technical review is ready to be merged. Thanks.

**Pull request merged**